lib/vcr.rb

```diff
@@ -107,6 +107,8 @@ def current_cassette
   # @option options :persist_with [Symbol] Which cassette persister to
   #  use. Defaults to :file_system. You can also register and use a
   #  custom persister.
+  # @option options :persister_options [Hash] Pass options to the
+  # persister specified in `persist_with`.
```

Comment:
If a reader were here - would they know anywhere in the docs that the option available to them was "downcase..."? Can we put it in here, to make it discoverable within the rendered API docs?

Reply:
(This is not a blocker to merging, I hasten to say.)

Reply:
How about something like this? (I'm not too familiar with how much formatting these API docs support so let me know if there's a better way:)

```suggestion
  #  persister specified in `persist_with`. Currently available options:
  #    - downcase_cassette_names: when `true`, names of cassettes will be
  #      normalized in lowercase before reading and writing, which can avoid
  #      confusion when using both case-sensitive and case-insensitive file
  #      systems.
```

Reply:
In order to find out if your edit looks good, you can run `yard`, the documentation tool, and then open doc/index.html, navigate to the rendered HTML, and see the rendering.

Reply:
The language is 👍 💯 though!

Reply:
Cool! I'll give `yard` a shot on Monday, I don't have access to my dev machine at the moment.

Reply:
I'll render a preview here, as a picture:

<img width="1323" alt="image" src="https://user-images.githubusercontent.com/211/75609688-15fea800-5b0b-11ea-8ecc-7918055f9b89.png">

Reply:
(This was after some minor fiddling. 

I was using the `yard server --reload` mode. Re-renders the viewed page on reload in the browser.)

Reply:
Looks good, thanks for the screenshot! I updated my original suggestion with your edits. I also added clarification that the option is for the file_system persister.